Framework/Algorithms/test/CalculatePlaczekSelfScatteringTest.h

```diff
@@ -156,7 +156,7 @@ class CalculatePlaczekSelfScatteringTest : public CxxTest::TestSuite {
     alg->setProperty("IncidentSpecta", IncidentSpecta);
     alg->setProperty("InputWorkspace", InputWorkspace);
     alg->setProperty("OutputWorkspace", "correction_ws");
-    TS_ASSERT_THROWS(alg->execute(), std::runtime_error)
+    TS_ASSERT_THROWS(alg->execute(), std::runtime_error &)
```

Comment:
Most places use `const exc_type &`. Can we use this here for consistency?

Reply:
I've gone through and added `const` to all of the changed locations for general consistency